test/cmd.js

```diff
@@ -115,7 +115,7 @@ describe('express(1)', function () {
       it('should create basic app', function (done) {
         run(ctx.dir, [], function (err, output) {
           if (err) return done(err)
-          assert.equal(parseCreatedFiles(output, ctx.dir).length, 17)
```

Comment:
I think there is to much discussion that needs to happen i.r.t to the `server.js` and `www` split. Can you please pull that out of this PR and into it's own PR for a discussion?